github-for-jira.sd.yml

```diff
@@ -43,7 +43,7 @@ links:
   source:
     url: git@bitbucket.org:atlassian/github-jira-integration.git
 
-cleanup: true
+cleanup: false
```

Comment:
Had to set to `false` so our service won't be automatically undeployed if we didn't deploy it for a few days.

Reply:
:scream:

Reply:
I'm pretty sure this is not enforced in production environments. Right? Right???? :)

Reply:
No, this is only for ddev, as far as I remember. Definitely not a thing in prod